spec/lib/secure_headers/headers/policy_management_spec.rb

```diff
@@ -183,7 +205,8 @@ module SecureHeaders
         non_default_source_additions = ContentSecurityPolicy::NON_FETCH_SOURCES.each_with_object({}) do |directive, hash|
           hash[directive] = %w("http://example.org)
         end
-        combined_config = ContentSecurityPolicy.combine_policies(Configuration.get.csp.to_h, non_default_source_additions)
+        default_policy = Configuration.send(:default_config)
```

Comment:
Yup..this makes a handful of tests uglier, but I don't like making private internal data more visible simply for the sake of tests.